### ISSUE TITLE
Tuple.toString fails with null elements

### ISSUE THREAD

**Issue by osi** [jlink/jqwik#187]
I'm storing a `null` value in a `Tuple`. When I print it as a string, I get a NPE.

```
import net.jqwik.api.Tuple;
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;

public class TupleToStringTest {
    @Test
    void shouldToString() {
        assertThat(Tuple.of("hello", null).toString()).isEqualTo("(hello, null)");
    }
}
```

```
java.lang.NullPointerException
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at net.jqwik.api.Tuple.itemsToString(Tuple.java:23)
	at net.jqwik.api.Tuple$Tuple2.toString(Tuple.java:181)
```

**Comment by jlink** [comment]
A single character difference for me, a whole lot difference for null values.

The sad thing is, that I missed the obvioud fuzzing property :-(

Bug fixed in https://github.com/jlink/jqwik/commit/15a8b3c375cbd1881c96cedb59cdd8983bdc3cec

Released to 1.5.2-SNAPSHOT

**Comment by osi** [comment]
heh no worries. it's sometimes that "simple seeming" code that has the hidden bug 😀
